CONTRIBUTING.adoc

```diff
@@ -0,0 +1,21 @@
+= Code formatting guidelines
+
+* The directory ./src/eclipse has two files for use with code formatting,
+`eclipse-code-formatter.xml` for the majority of the code formatting rules and `eclipse.importorder`
+to order the import statements.
+
+* In Eclipse you import these files by navigating `Windows -> Preferences` and then the menu items
+`Preferences > Java > Code Style > Formatter` and `Preferences > Java > Code Style >
+Organize Imports` respectively.
+
+* In IntelliJ IDEA, install the plugin `Eclipse Code Formatter`.  You can find it by searching in
+"Browse Repositories", under `Settings > Plugins` within IDEA (Once installed, you will need to
+reboot IDEA for it to take effect).
+Then navigate to `IntelliJ IDEA > Preferences` and select the Eclipse Code Formatter.
```

Comment:
On Linux, for me it's `IntelliJ IDEA > Settings > Other Settings > Eclipse Code Formatter`. I think on Mac it's "Preferences" instead of "Settings"